tests/mail/tests.py

```diff
@@ -913,6 +913,26 @@ def test_attach_non_utf8_text_as_bytes(self):
         self.assertEqual(content, b"\xff")
         self.assertEqual(mimetype, "application/octet-stream")
 
+    def test_attach_8bit_rfc822_message(self):
+        """
+        Test that attaching a message containing non-ASCII characters does not
+        raise a UnicodeEncodeError (#36119).
+        """
+        att = dedent(
```

Comment:
I would use `attachment` instead of `att` (an unusual abbreviation with no precedent in this file).